tests/test_values.py

```diff
@@ -96,8 +95,7 @@ def test_space_in_project_slug(cookies):
     os.chdir(str(project.project))
 
     try:
-        sh.python(['setup.py', 'install'])
-        sh.python(['setup.py', 'build_sphinx'])
+        sh.sphinx_build(["-a", "-b", "html", "docs", "docs/_build"])
```

Comment:
Can we make this work with `make html`, since that is what the instructions say. See https://github.com/NLeSC/python-template/issues/102